web/ui/react-app/src/App.css

```diff
@@ -10,6 +10,24 @@ button.classic-ui-btn {
     margin-bottom: 20px;
 }
 
+.checkbox{
+    display: inline-flex;
+    align-items: center;
+    cursor: pointer;
+    margin: 0 0 13px 3px;
+}
+
+.checkbox.checked {
+    color: #286090
```

Comment:
I think this color doesn't fit anymore to the bootstrap version that the new UI is using. Also, I added a "Return to classic UI" at the top right and in the old UI made it share the same row as the query history checkbox. Could we adjust colors (same as "Return to classic UI" link), font sizes, and positioning to be more like that? Dirty mockup using float left/right for query history + back link in a parent container:

![query_history_mockup](https://user-images.githubusercontent.com/538008/67288958-1c866e80-f4de-11e9-966b-0d56e507f8d1.png)

Reply:
Yep, will make those adjustments, thanks